espanol/granhermano/granhermano.md

```diff
@@ -0,0 +1,5 @@
+La guerra es la paz. La libertad es la esclavitud. La ignorancia es la fuerza 
+
+Lo más característico de la vida moderna no era su crueldad ni su inseguridad, sino sencillamente su vaciedad, su absoluta falta de contenido 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.